app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -37,12 +39,14 @@ import dagger.android.support.HasSupportFragmentInjector
 import kotlinx.android.synthetic.main.activity_main.*
 import kotlinx.android.synthetic.main.app_bar.*
 import kotlinx.android.synthetic.main.nav_header.view.*
+import java.util.Locale
 import javax.inject.Inject
 import android.app.NotificationManager
-import android.content.Context
 
 
 private const val CURRENT_STATE = "current_state"
+private const val SETTING = "Settings"
```

Comment:
Follow the "pattern" we are using in the codebase. It should be `settings` instead of `Settings` and `my_lang` instead of `My_Lang`. 

Better, define something like `private const val APP_LANGUAGE = "app_language"`